config/checkstyle_checks.xml

```diff
@@ -709,6 +709,9 @@
       <property name="tokens" value="METHOD_REF"/>
     </module>
     <module name="NoWhitespaceBefore"/>
+    <module name="NoWhitespaceBefore">
+      <property name="tokens" value="ELLIPSIS"/>
+    </module>
```

Comment:
please update default NoWhitespaceBefore to have no `ELLIPSIS` in tokens list.

Reply:
done